### ISSUE TITLE
what network qos really do ?  only set cgroup net class id?

### ISSUE THREAD

**Issue by gaopeiliang** [kubewharf/katalyst-core#303]
I have see 0.3 release an network qos plugin ,,  is there any doc about it ? 

we just see set cgroup net class id,,  but where tc policy ready do ?  we should manager TC policy  manual ?

**Comment by waynepeking348** [comment]
> I have see 0.3 release an network qos plugin ,, is there any doc about it ?
> 
> we just see set cgroup net class id,, but where tc policy ready do ? we should manager TC policy manual ?

yeah, we will depend on edt/tc or something like that to perform the bandwidth control logic. also, network plugin will be responsible for allocating nic with numa affinity (or any affinity level)

**Comment by gaopeiliang** [comment]
> > I have see 0.3 release an network qos plugin ,, is there any doc about it ?
> > we just see set cgroup net class id,, but where tc policy ready do ? we should manager TC policy manual ?
> 
> yeah, we will depend on edt/tc or something like that to perform the bandwidth control logic. also, network plugin will be responsible for allocating nic with numa affinity (or any affinity level)

ok ,,  another question , what means "allocating nic" ?    there many NIC on host and should move netns like sriov ?   should use special CNI plugin ?    hostnetwork mode Container will see all nic always ....
